tests/python-gpu/load_pickle.py

```diff
@@ -0,0 +1,20 @@
+'''Loading a pickled model generated by test_pickling.py'''
+import pickle
+import unittest
+import os
+import xgboost as xgb
+import sys
+
+sys.path.append("tests/python")
```

Comment:
This is not needed, because tests are run after XGBoost is installed into the local Python environment.

Reply:
@hcho3 Those tests aren't installed.  I don't think I can import `from test_pickling import build_dataset, model_path` without proper sys path.

Reply:
@trivialfis Ah never mind then.